components/ui/AppDataTable.vue

```diff
@@ -30,29 +43,7 @@ defineProps<Props>();
 </style>
 
 <style lang="scss">
-@use "~/assets/scss/carbon.scss";
-@use "~/assets/scss/helpers/index.scss" as qiskit;
-
-.bx--data-table th {
-  color: carbon.$black-100;
-  background-color: qiskit.$background-color-light;
-
-  // TODO: remove this override once issue is fixed
-  // https://github.com/carbon-design-system/carbon-components-vue/issues/1161#issuecomment-815689534
-  &[aria-sort] {
-    padding: carbon.$spacing-05;
-  }
-}
-
-.bx--data-table tbody tr td,
-.bx--data-table tbody tr:hover td {
-  background-color: qiskit.$background-color-white;
-  border-top: none;
-  border-bottom: 1px solid qiskit.$border-color;
-}
-
-.bx--data-table li {
-  list-style-type: disc;
-  margin-left: carbon.$spacing-06;
+bx-table-body {
+  background-color: white;
```

Comment:
Do we want to use a carbon token here?

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3017/commits/7f3e1eaec186f200b40514a9c3fa5c1021925e92